app/Notifications/UserAccessBranchNotification.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Notification;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Messages\MailMessage;
+
+class UserAccessBranchNotification extends Notification
+{
+    use Queueable;
+
+    protected $user;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct($user, $branches)
+    {
+        $this->user = $user;
+        $this->branches = $branches;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param  mixed  $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+                    ->greeting('Hello! ' . $this->user->name)
+                    ->subject('Access to branch update')
+                    ->line($this->branches. 'has granted access to your account '. $this->user->email. '.');
```

Comment:
$this->user->email should have brackets as well

to your account (' . $this->user->email . ').');

Reply:
that's why I change the array to a string using implode and pass it to the class so it will be something like Parcelhub DU, ParcelHub perlis has granted access to your account cause admin can select multiple option.

Reply:
This email will only be sent when admin try to register user into 1 branch right? So this email should only contain that 1 branch.

I know it's possible to have multiple branches for 1 user, but it doesn't concern our email here cause at the point of time where admin adds the user there's only 1 branch,